aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cu

```diff
@@ -1220,7 +1220,7 @@ inline bool jagged_dense_dense_elementwise_jagged_output_matches_opt(
   // MI100 has independent shared mem and L1
   int used_shared_kb = shared_kb;
 #endif
-  int used_shared_bytes = used_shared_kb << 10;
+  auto used_shared_bytes = static_cast<size_t>(used_shared_kb << 10);
```

Comment:
Can you do `size_t used_shared_bytes = used_shared_kb << 10;` ?

Reply:
It will trigger the same warning that one is trying to implicitly cast signed to unsigned value. (used_shared_kb is positive int )